src/components/PageTitle/index.tsx

```diff
@@ -1,14 +1,37 @@
 import React, {FC} from 'react';
 import {Helmet} from 'react-helmet-async';
 
-interface ComponentProps {
+export interface PageTitleProps {
   title: string;
+  ogDescription?: string;
+  ogImageUrl?: string;
+  ogTitle?: string;
+  ogType?: 'website' | 'article' | 'profile' | 'book';
+  ogUrl?: string;
 }
 
-const PageTitle: FC<ComponentProps> = ({title}) => {
+export const defaultOgDescription =
+  'Learn to code, collaborate on projects, gain experience, build a community, and earn coins by contributing.';
+export const defaultOgImageUrl = 'https://www.thenewboston.com/logo192.png';
+export const defaultOgType = 'website';
+export const defaultOgUrl = 'https://www.thenewboston.com/';
```

Comment:
Not really sure about this, but do think it would be good to have an object instead of different variables?

```
const defaultOg = {
  description: 'Hello',
  imageUrl: 'hello.jpg'
}
```

Reply:
yeah sure will do so